src/Features/CSharp/Portable/InitializeParameter/InitializeParameterHelpers.cs

```diff
@@ -19,6 +20,11 @@ public static bool IsFunctionDeclaration(SyntaxNode node)
             || node is LocalFunctionStatementSyntax
             || node is AnonymousFunctionExpressionSyntax;
 
+        public static IReadOnlyList<SyntaxNode> GetParameters(SyntaxNode node)
+        {
+            return ((BaseMethodDeclarationSyntax)node).ParameterList.Parameters;
```

Comment:
this might crash because `node` can be something other than `BaseMethodDeclarationSyntax`